### ISSUE TITLE
Some tests in test.test_screen.ScreenTest are failing

### ISSUE THREAD

**Issue by virtuallight** [lordmauve/pgzero#96]
OS: Mac 10.13.6
Python: 3.7.0
PyGame: 1.9.4

```
======================================================================
FAIL: test_blit_name (test.test_screen.ScreenTest)
screen.blit() accepts an image name instead of a Surface.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mateuszjankowski/Dropbox/projects/programming/side-projects/pygame/pgzero/test/test_screen.py", line 35, in test_blit_name
    self.assertImagesEqual(surf, images.expected_alien_blit)
  File "/Users/mateuszjankowski/Dropbox/projects/programming/side-projects/pygame/pgzero/test/test_screen.py", line 25, in assertImagesEqual
    raise AssertionError("Images differ")
AssertionError: Images differ

======================================================================
FAIL: test_blit_surf (test.test_screen.ScreenTest)
We can blit a surface to the screen.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mateuszjankowski/Dropbox/projects/programming/side-projects/pygame/pgzero/test/test_screen.py", line 30, in test_blit_surf
    self.assertImagesEqual(surf, images.expected_alien_blit)
  File "/Users/mateuszjankowski/Dropbox/projects/programming/side-projects/pygame/pgzero/test/test_screen.py", line 25, in assertImagesEqual
    raise AssertionError("Images differ")
AssertionError: Images differ

----------------------------------------------------------------------
```

**Comment by virtuallight** [comment]
Working on it.

**Comment by lordmauve** [comment]
Fixed in #102
